src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,86 @@
+@use "../utils/mixins" as *;
+
+.menu {
+  z-index: 2;
+  background-color: #e5e6e8;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  min-height: 100vh;
+  padding: 0 20px;
+  transform: translateX(-100%);
+  transition: transform 0.3s ease-in-out;
+  align-content: start;
+
+  &__navigation {
+    grid-template-columns: 32px;
+    height: 32px;
+  }
+
+  &:target {
+    transform: translateX(0);
+  }
+
+  @include for-tablet() {
+    padding: 0 72px;
+  }
+
+  @include for-desktop() {
+    padding: 0 120px;
+  }
+
+  &__paragraph {
+    font-family: "Space Grotesk", sans-serif;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 22px;
+    letter-spacing: 2px;
+    text-transform: uppercase;
+    color: #292929;
+
+    &:first-of-type {
+      padding-bottom: 16px;
+    }
+
+    &:last-of-type {
+      border-bottom: 1px solid #292929;
+      width: 142px;
```

Comment:
The '.menu__paragraph:last-of-type' selector has a fixed width of 142px. This fixed width might not be responsive on all screen sizes. Consider using a `max-width` here